src/components/notes/List/NoteRow.jsx

```diff
@@ -83,17 +83,22 @@ const NoteRow = ({ note, f, t, client }) => {
         >
           <Icon icon={NoteIcon} size={32} className="u-mr-1 u-flex-shrink-0" />
 
-          <div className="u-flex-grow-1">
-            <span className="u-charcoalGrey u-ellipsis">{filename}</span>
-
-            <WithBreakpoints hideOn={Breakpoints.Mobile}>
+          <WithBreakpoints hideOn={Breakpoints.Mobile}>
+            <div className="u-flex-grow-1">
+              <span className="u-charcoalGrey u-ellipsis">{filename}</span>
               <span className="u-ellipsis">{extension}</span>
-            </WithBreakpoints>
+            </div>
+          </WithBreakpoints>
 
-            <WithBreakpoints showOn={Breakpoints.Mobile}>
+          <WithBreakpoints showOn={Breakpoints.Mobile}>
+            <div
+              className="u-flex u-flex-column u-flex-grow-1"
+              style={{ minWidth: 0 }}
```

Comment:
minWidth needed to block div to take all space see https://stackoverflow.com/a/66689926